app/src/main/java/org/sopt/santamanitto/chat/overview/ChatOverviewViewHolder.kt

```diff
@@ -0,0 +1,58 @@
+package org.sopt.santamanitto.chat.overview
+
+
+import androidx.core.view.isVisible
+import androidx.recyclerview.widget.RecyclerView
+import org.sopt.santamanitto.R
+import org.sopt.santamanitto.databinding.ItemChatOverviewBinding
+import org.sopt.santamanitto.util.TimeUtil.convertToElapsedTime
+import org.sopt.santamanitto.util.TimeUtil.isExpired
+import java.util.Locale
+
+class ChatOverviewViewHolder(
+    val binding: ItemChatOverviewBinding,
+    val itemClick: (String) -> Unit,
+    val itemLongClick: (String) -> Unit
+) : RecyclerView.ViewHolder(binding.root) {
+
+    fun onBind(item: ChatItemModel) {
+        with(binding) {
+            textviewOverviewItemTitle.text = item.roomName
+
+            textviewOverviewItemContent.text = if (item.lastContent.isNullOrBlank()) {
+                "${item.opponentName}에게 쪽지를 보내보자!"
+            } else {
+                item.lastContent
+            }
+
+            textviewOverviewItemTime.text = if (item.lastMessageAt.isNullOrBlank()) {
+                ""
+            } else {
+                convertToElapsedTime(item.lastMessageAt)
+            }
```

Comment:
takeIf 같은 문법으로 좀 더 간결하게 쓸 수 있을 것 같아요! 

```suggestion
textviewOverviewItemContent.text = item.lastContent.takeIf { !it.isNullOrBlank() }
    ?: "${item.opponentName}에게 쪽지를 보내보자!"
```